test/e2e/data/infrastructure-docker/main/clusterclass-quick-start.yaml

```diff
@@ -126,44 +126,6 @@ spec:
         valueFrom:
           template: |
             imageRepository: {{ .lbImageRepository }}
```

Comment:
Should we now also be able to drop the corresponding handling for MachinePools in the upgrades-cgroupfs cluster-template?  (folder cluster-template-upgrades-cgroupfs + the if/else in cluster_upgrade_test.go)

Reply:
I can test this, I just need to catch up with this edge case

Reply:
tl;dr is basically that we have this because MachinePools are not supported in ClusterClass, so we needed more than the ClusterClass to set cgroup driver correctly

Reply:
Looks good!